[
  {
    "url": "https://api.github.com/repos/grpc/grpc/issues/comments/147047347",
    "html_url": "https://github.com/grpc/grpc/issues/3748#issuecomment-147047347",
    "issue_url": "https://api.github.com/repos/grpc/grpc/issues/3748",
    "id": 147047347,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE0NzA0NzM0Nw==",
    "user": {
      "login": "mkfire",
      "id": 5850302,
      "node_id": "MDQ6VXNlcjU4NTAzMDI=",
      "avatar_url": "https://avatars3.githubusercontent.com/u/5850302?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/mkfire",
      "html_url": "https://github.com/mkfire",
      "followers_url": "https://api.github.com/users/mkfire/followers",
      "following_url": "https://api.github.com/users/mkfire/following{/other_user}",
      "gists_url": "https://api.github.com/users/mkfire/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/mkfire/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/mkfire/subscriptions",
      "organizations_url": "https://api.github.com/users/mkfire/orgs",
      "repos_url": "https://api.github.com/users/mkfire/repos",
      "events_url": "https://api.github.com/users/mkfire/events{/privacy}",
      "received_events_url": "https://api.github.com/users/mkfire/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-10-10T06:58:01Z",
    "updated_at": "2015-10-10T06:58:40Z",
    "author_association": "NONE",
    "body": "I have the same issue with crash in AsyncNextInternal()\n"
  },
  {
    "url": "https://api.github.com/repos/grpc/grpc/issues/comments/158793585",
    "html_url": "https://github.com/grpc/grpc/issues/3748#issuecomment-158793585",
    "issue_url": "https://api.github.com/repos/grpc/grpc/issues/3748",
    "id": 158793585,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE1ODc5MzU4NQ==",
    "user": {
      "login": "ohad7",
      "id": 9012046,
      "node_id": "MDQ6VXNlcjkwMTIwNDY=",
      "avatar_url": "https://avatars2.githubusercontent.com/u/9012046?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ohad7",
      "html_url": "https://github.com/ohad7",
      "followers_url": "https://api.github.com/users/ohad7/followers",
      "following_url": "https://api.github.com/users/ohad7/following{/other_user}",
      "gists_url": "https://api.github.com/users/ohad7/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ohad7/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ohad7/subscriptions",
      "organizations_url": "https://api.github.com/users/ohad7/orgs",
      "repos_url": "https://api.github.com/users/ohad7/repos",
      "events_url": "https://api.github.com/users/ohad7/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ohad7/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-11-22T19:55:46Z",
    "updated_at": "2015-11-22T19:55:46Z",
    "author_association": "NONE",
    "body": "I'm sorry to say, but we've been working with GRPC for 6 months now in production, and as a result of various stability issues such as the above are forced to go back to REST based HTTP/1.1. \n\nThe combination of an iOS application having multiple crashes, general lack of support and very poor success rate (GRPC success rate of calls in a real world mobile internet environment is around 40%) does not leave us much of choice.\n"
  },
  {
    "url": "https://api.github.com/repos/grpc/grpc/issues/comments/169101365",
    "html_url": "https://github.com/grpc/grpc/issues/3748#issuecomment-169101365",
    "issue_url": "https://api.github.com/repos/grpc/grpc/issues/3748",
    "id": 169101365,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE2OTEwMTM2NQ==",
    "user": {
      "login": "yang-g",
      "id": 10122250,
      "node_id": "MDQ6VXNlcjEwMTIyMjUw",
      "avatar_url": "https://avatars0.githubusercontent.com/u/10122250?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/yang-g",
      "html_url": "https://github.com/yang-g",
      "followers_url": "https://api.github.com/users/yang-g/followers",
      "following_url": "https://api.github.com/users/yang-g/following{/other_user}",
      "gists_url": "https://api.github.com/users/yang-g/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/yang-g/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/yang-g/subscriptions",
      "organizations_url": "https://api.github.com/users/yang-g/orgs",
      "repos_url": "https://api.github.com/users/yang-g/repos",
      "events_url": "https://api.github.com/users/yang-g/events{/privacy}",
      "received_events_url": "https://api.github.com/users/yang-g/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-01-05T19:14:48Z",
    "updated_at": "2016-01-05T19:14:48Z",
    "author_association": "MEMBER",
    "body": "I am sorry to hear that. We are trying to add test coverage and improve stability while heading to GA.\n\nCrash 1 was explicitly marked \"not implemented\" in the code and should have been fixed by #4097.\n\nCrash 2 and 3 are not immediately clear to me from the stacktrace provided. Crash 2 could be caused by some assert failures somewhere deep under grpc_fd_orphan, but the stacktrace is not enough to tell where. Crash 3 seems to be an assert failure for metadata element ref-counting, and it should never happen. There was a code change there but it should not fix the crash. \n\nIt would be nice to have a way to reproduce it for us to investigate more.\n\nThanks.\n"
  },
  {
    "url": "https://api.github.com/repos/grpc/grpc/issues/comments/171906531",
    "html_url": "https://github.com/grpc/grpc/issues/3748#issuecomment-171906531",
    "issue_url": "https://api.github.com/repos/grpc/grpc/issues/3748",
    "id": 171906531,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE3MTkwNjUzMQ==",
    "user": {
      "login": "mackross",
      "id": 247049,
      "node_id": "MDQ6VXNlcjI0NzA0OQ==",
      "avatar_url": "https://avatars0.githubusercontent.com/u/247049?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/mackross",
      "html_url": "https://github.com/mackross",
      "followers_url": "https://api.github.com/users/mackross/followers",
      "following_url": "https://api.github.com/users/mackross/following{/other_user}",
      "gists_url": "https://api.github.com/users/mackross/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/mackross/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/mackross/subscriptions",
      "organizations_url": "https://api.github.com/users/mackross/orgs",
      "repos_url": "https://api.github.com/users/mackross/repos",
      "events_url": "https://api.github.com/users/mackross/events{/privacy}",
      "received_events_url": "https://api.github.com/users/mackross/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-01-15T09:04:20Z",
    "updated_at": "2016-01-15T09:04:20Z",
    "author_association": "NONE",
    "body": "Connectivity will continue to be an issue for iOS until CFStream or a preflight request is used to power up radio gear #4687.\n"
  }
]
